theorems/T000641.md

```diff
@@ -0,0 +1,11 @@
+---
+uid: T000641
+if:
+  P000205: true
+then:
+  P000107: true
+refs:
+- doi: 10.1090/s0002-9939-99-04839-x
+  name: Cut-Point Spaces
+---
+Theorem 3.7 in {{doi:10.1090/s0002-9939-99-04839-x}} states that a cut point space has infinitely many closed points.
```

Comment:
Saying "states that" seems that it is just stated, but not necessily shown/proved.
How about saying instead: "By theorem ..., a cut point space ..."